scripts/debug_tools/prepare_compile_cmd.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
```

Comment:
Please add license.

Reply:
Is this worth to create a separate script, there is only one method here? The same question applies to `prepare_compiler_includes.py` and `prepare_compiler_target.py`

Reply:
During debugging each of the compile_cmd, compiler_includes, compiler_target files might be used separately, so I think it is worth to have them as separate files.